ui/design-system-core/src/components/Stepper/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { tx, cx } from '@twind/core';
+
+export interface IStepIndicatorProps {
+  activeIndex: number;
+  stepLabels: string[];
+}
+
+const IncompletedStepIcon = ({ color = 'grey5', darkColor = 'grey6' }) => {
```

Comment:
extract the icons in separate files in the akasha-icons folder of the icon component in design-system-core

Reply:
I have updated it :)